firestarter/workflows/build_images/providers/registries/github.py

```diff
@@ -0,0 +1,9 @@
+import os
+from .base import RegistryAuth, RegistryProvider
+
+
+class GithubRegistryAuth(RegistryProvider):
+
+    def get_registry_auth(self):
+        # split the username and password
+        return RegistryAuth(username=os.environ['GITHUB_ACTOR'], token=os.environ['GH_TOKEN'])
```

Comment:
Why not use directly the env var GITHUB_TOKEN 🤔? 

Are you thinking in support also a PAT so you can use GH_TOKEN as an intermediate? 
```
GH_TOKEN = secrets.PAT || github.token 
```

Reply:
It does not exist in the env, so we have to pass it explicitly. The `||` statement is a great approach.